### PR TITLE
#104 Add a install to script recoder extension.

### PR DESCRIPTION
**Pull request by GwonGisoo**
- Create package to '.ctx' from ngrinder-recorder
- Deploy a 'ngrinder-recorder-1.0.crx' at 'ngrinder-controller/src/main/webapp/crx/'
- Add the guide for ngrinder recorder install in ngrinder-controller
- Add the "README.md" for ngrinder-recorde guide.
- 'ngrinder-recorder.pem' is automatically created during packaging to 'crx' by chrome browser

### CODE REVIEW

File: ngrinder-controller/src/main/resources/messages_cn.properties [hunk 1]
```diff
@@ -65,6 +65,7 @@ navigator.dropDown.announcement=\u516c\u544a\u7f16\u8f91\u5668
 navigator.dropDown.downloadAgent=\u4e0b\u8f7d\u4ee3\u7406
 navigator.dropDown.downloadPrivateAgent=\u4e0b\u8f7d\u4ee3\u7406
 navigator.dropDown.downloadMonitor=\u4E0B\u8F7D\u76D1\u63A7
+navigator.dropDown.downloadRecorder=\u4E0B\u8F7D\u5F55\u5236\u63D2\u4EF6
```
Comment (GwonGisoo) [inline ngrinder-controller/src/main/resources/messages_cn.properties]: @chengaomin Thanks for the translate ^^

File: ngrinder-controller/src/main/webapp/WEB-INF/ftl/common/recorder_guide_modal.ftl [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<div class="modal hide fade" id="recorder_guide_modal" tabindex="-1" role="dialog"  aria-labelledby="myModalLabel" aria-hidden="true">
+	<div class="modal-header">
+		<button type="button" class="close" data-dismiss="modal" aria-hidden="true">x</button>
+		<h4>nGrinder Recorder Install Guide</h4>
+	</div>
+	<div class="modal-body">
+		<div>
+			<p>Follow below step in Chrome browser:</p>
```
Comment (junoyoon) [inline ngrinder-controller/src/main/webapp/WEB-INF/ftl/common/recorder_guide_modal.ftl]: steps

File: ngrinder-controller/src/main/webapp/WEB-INF/ftl/common/recorder_guide_modal.ftl [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+			<ol>
+				<li>
+					nGrinder Recorder <a href="${req.getContextPath()}/crx/ngrinder-recorder-1.0.crx">download here</a>
```
Comment (junoyoon) [inline ngrinder-controller/src/main/webapp/WEB-INF/ftl/common/recorder_guide_modal.ftl]: Download  <a href="${req.getContextPath()}/crx/ngrinder-recorder-1.0.crx">nGrinder Recorder</a>/

File: ngrinder-controller/src/main/webapp/WEB-INF/ftl/common/recorder_guide_modal.ftl [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+				</li>
+				<li>
+					Move to the Extensions management page by this URL: <b>chrome://extensions</b><br>
+					(other way : Chrome menu -> Tools -> Extensions)
+				</li>
+				<li>
+					Drags the <b>ngrinder-recorder.crx</b> file into the Extensions management page
```
Comment (junoyoon) [inline ngrinder-controller/src/main/webapp/WEB-INF/ftl/common/recorder_guide_modal.ftl]: Extensions ==> extension

File: ngrinder-controller/src/main/webapp/WEB-INF/ftl/common/recorder_guide_modal.ftl [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+				</li>
+				<li>
+					Accept in installing
```
Comment (junoyoon) [inline ngrinder-controller/src/main/webapp/WEB-INF/ftl/common/recorder_guide_modal.ftl]: ??

File: ngrinder-controller/src/main/webapp/WEB-INF/ftl/common/recorder_guide_modal.ftl [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+				</li>
+				<li>
+					If completed, you will be able to see nGrinder Recorder Extension icon!
+				</li>
+			</ol>
+			<p>If you want a how to usage, <a href="https://github.com/naver/ngrinder/tree/master/ngrinder-controller#ngrinder-recorder-chrome-extension">show nGrinder Recorder READMD.md</a></p>
```
Comment (junoyoon) [inline ngrinder-controller/src/main/webapp/WEB-INF/ftl/common/recorder_guide_modal.ftl]: If you want to know how to use....

File: ngrinder-controller/src/main/webapp/WEB-INF/ftl/common/recorder_guide_modal.ftl (hunk 5 above)
Comment (GwonGisoo) [inline ngrinder-controller/src/main/webapp/WEB-INF/ftl/common/recorder_guide_modal.ftl]: It change to 'Allow for install' ?

Comment (junoyoon) [inline ngrinder-controller/src/main/webapp/WEB-INF/ftl/common/recorder_guide_modal.ftl]: Allow to install local extension.

File: ngrinder-controller/src/main/webapp/WEB-INF/ftl/common/recorder_guide_modal.ftl (hunk 6 above)
Comment (junoyoon) [inline ngrinder-controller/src/main/webapp/WEB-INF/ftl/common/recorder_guide_modal.ftl]: Extensions management page => extensions management page.

**Pull request merged**